build.sbt

```diff
@@ -2,13 +2,18 @@ organization := "com.github.swagger-akka-http"
 
 name := "swagger-akka-http"
 
-ThisBuild / scalaVersion := "2.13.6"
-
 val swaggerVersion = "2.1.10"
 val akkaVersion = "2.6.16"
 val akkaHttpVersion = "10.2.6"
 val jacksonVersion = "2.12.5"
 val slf4jVersion = "1.7.32"
+val scala213 = "2.13.6"
+
+ThisBuild / scalaVersion := scala213
+ThisBuild / crossScalaVersions := Seq(scala213, "2.12.15")
+
+// NOTE: Once akka updates to 1.0.0 of scala-java8-compat, remove this
+ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-java8-compat" % "always"
```

Comment:
This is necessary for the time being to avoid
```
[error]         * org.scala-lang.modules:scala-java8-compat_2.12:1.0.0 (early-semver) is selected over 0.8.0
[error]             +- com.github.swagger-akka-http:swagger-akka-http_2.12:2.5.0+8-df2f6833+20210920-1240-SNAPSHOT (depends on 1.0.0)
[error]             +- com.typesafe.akka:akka-actor_2.12:2.6.16           (depends on 0.8.0)
```
This is happening because 1.0.0 starts setting `early-semver` meaning that it won't just evict 0.8.0 without complaining and erroring. This should be quite safe to do here as looking at the release notes https://github.com/scala/scala-java8-compat/releases the reason a 1.0.0 release was cut was due to adding in the actually version scheme, not any other breaking changes. So for now I put this in here and then once Akka updates to 1.0.0 we can just remove this.